ee/bin/docker-ch-dev-web

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-./ee/bin/docker-ch-dev-backend & ./bin/docker-frontend
+./ee/bin/docker-ch-dev-backend
```

Comment:
Not a good move. The `arm64:ch-dev:start` script contains

`CH_WEB_SCRIPT=./ee/bin/docker-ch-dev-backend`

... which overrides things and runs the `-backend` script directly instead of this `-dev-web` script.

Who knows what will this break.